### ISSUE TITLE
Accessibility: Change bold to strong in join-us.html

### ISSUE THREAD

**Issue by macho-catt** [hackforla/website#2431]
### Prerequisite
Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
To be WCAG compliant and ensure the same experience to all users, we want to change b tags to strong tags.

### Action Items
- [x] Change all the b tags to strong tags within `pages/join-us.html`
- [x] **Line 20**

From:
```
Hack for LA is a project of <b>Code for America</b> and is its official
```
To:
```
Hack for LA is a project of <strong>Code for America</strong> and is its official
```
- [x] **Line 43**

From:
```
Connect with our leadership team on the <b>#admin</b> <a href="https://hackforla.slack.com/messages/admin"
```
To:
```
Connect with our leadership team on the <strong>#admin</strong> <a href="https://hackforla.slack.com/messages/admin"
```

### Resources/Instructions
- [Article on why we should use semantic tags like strong and em](https://help.siteimprove.com/support/solutions/articles/80000448460-accessibility-bold-vs-strong-and-italic-vs-emphasis)
- [Affected webpage](https://www.hackforla.org/join)
- Directory to find the page in once you have it in your IDE: `pages/join-us.html`
- Click on the filename below to see the source file code:
https://github.com/hackforla/website/blob/e29b4b2d474c012dfba681f1342d8cfcaa67039f/pages/join-us.html#L20
https://github.com/hackforla/website/blob/e29b4b2d474c012dfba681f1342d8cfcaa67039f/pages/join-us.html#L43

**Comment by doboianh** [comment]
Can i do it

**Comment by macho-catt** [comment]
hi @doboianh. Thank you for your interest in contributing to our organization. If you would like to contribute to our repository, you need to join our organization first. To join, please check this [website](https://www.hackforla.org/getting-started).

Joining involves being onboarded to our team, and following the contributing document. We only allow volunteers to work and fix one issue at a time to give other volunteers the opportunity for good first issues. After you've been onboarded, you can claim a single issue and open a PR to have it reviewed and merge.

After you've joined the team, I will be happy to assign you an issue and review your PRs. Thanks!

**Comment by muditchoudhary** [comment]
Hey 👋 I would like to work on this issue. I've just finished reading the contributing guidelines and have some questions to ask : 
- I don't know about Docker. Can I do this issue ?
- Is it optional or compulsory to install/use Docker? Btw I'm happy to learn it as I go ahead.
- Could you tell what could be the possible size of this repository?
-  Also I've found some little typos in the contributing guidelines. Should I raise a issue?

Thank You! 😄

**Comment by macho-catt** [comment]
@brightmudit you would need to join the organization first to be able to contribute to this repository. Please refer to this [website](https://www.hackforla.org/getting-started) on how to do that.

Additionally, you would need to install and use docker to run a local version of the website on your environment.

**Comment by muditchoudhary** [comment]
Ok, I've sign up for the next on-boarding session.

Thank You.
